lib/server/ServerAjax.js

```diff
@@ -57,7 +57,14 @@ var ServerAjax = {
                         throw xhr;
                     }
 
-                    var data = JSON.parse(bodyJSON);
+                    var data;
+                    var type = responseArgs[0].headers["content-type"];
+
+                    if (type === 'text/javascript;charset=UTF-8') {
```

Comment:
i think we should change some of the naming - bodyJSON -> responseBody - so that we can better understand what's going on. also we should probably use the same criteria that backbone.nativeajax uses - https://github.com/akre54/Backbone.NativeAjax/blob/master/backbone.nativeajax.js#L26. so JSON.parse when the content-type is application/json